### DISCUSSION TITLE
[Maxwell2D] skew model

### DISCUSSION THREAD

**Discussion by mehs1006** [ansys/pyaedt#3434] [Q&A] [answered]
Hi,

In the Maxwell2D module, there's a function called "skew model".
I wonder if there's a command in PyAEDT that can make a skew model.
Thanks.

**Comment by deepak-kr1592** [comment]
Hello @mehs1006 
I am also looking for it. So far , I did not find the function in pyAEDT to set skew model. I don"t know where to make a request to add any missing functions. :(

**Reply by mehs1006** [reply]
Hi @deepak-kr1592 ,
It's good to see someone has the same need.
I couldn't find it so far.
I will request a pull for this function.

**Reply by deepak-kr1592** [reply]
Looks like they added a skew function
https://aedt.docs.pyansys.com/version/stable/API/_autosummary/pyaedt.maxwell.Maxwell2d.apply_skew.html

Did you request for this function?

**Comment by MaxJPRey** [comment]
Hi@mehs1006 @deepak-kr1592
If you install the latest version of PyAEDT, you should be able to work with skew model in Maxwell 2D.

Please let us know if you face any issue with the feature.
Thanks.

**Reply by mehs1006** [reply, marked as answer]
Hi @MaxJPRey,
Thank you.
